index.bs

```diff
@@ -1997,10 +1997,10 @@ template:
     the [=public key credential=] to be attested, the [=authenticator data=] structure containing the <dfn>authenticator data
     for the attestation</dfn>, and the [=hash of the serialized client data=].
 
-- <dfn>Verification procedures</dfn>:
-    The procedure for verifying an [=attestation statement=], which takes as inputs the [=authenticator data=] structure
-    containing the <dfn>authenticator data claimed to have been used for the attestation</dfn> and the [=hash of the serialized
-    client data=], and returns either:
+- <dfn>Verification procedure</dfn>:
+    The procedure for verifying an [=attestation statement=], which takes as [=verification procedure inputs|inputs=] the
```

Comment:
s/[=verification procedure inputs|inputs=]/<dfn>verification procedure inputs</dfn>/

Reply:
See above, I kept the `<dfn>verification procedure inputs</dfn>` but inlined it in the existing paragraph. Does that look good or should I cut it completely?

Reply:
looks good!